### PR TITLE
handle multiple DOIs

### PR DESCRIPTION
**Pull request by maxulysse**
## PR checklist

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - For modules:
    - [ ] `nf-core modules test <MODULE> --profile docker`
    - [ ] `nf-core modules test <MODULE> --profile singularity`
    - [ ] `nf-core modules test <MODULE> --profile conda`
  - For subworkflows:
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile docker`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile singularity`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile conda`

### CODE REVIEW

**Comment by maxulysse** [comment]
On the terminal in sarek:

```
If you use nf-core/sarek for your analysis please cite:

* The pipeline
  https://doi.org/10.12688/f1000research.16665.2
  https://doi.org/10.1093/nargab/lqae031
  https://doi.org/10.5281/zenodo.3476425
```

instead of:
```
If you use nf-core/sarek for your analysis please cite:

* The pipeline
  10.12688/f1000research.16665.2, 10.1093/nargab/lqae031, 10.5281/zenodo.3476425
```

This doesn't change a single thing for rnaseq

**Review by nvnieuwk — APPROVED** [review]
LGTM, but maybe wait for a second review on this one :p

**Review by mashehu — APPROVED** [review]

**Pull request merged**
